src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -80,7 +86,7 @@ public function getIssueUrl($id)
             $this->configuration['repo_domain_url'],
             $this->getUsername(),
             $this->getRepository(),
-            $this->getIssue($id)['iid']
+            ($id instanceof Issue)?$id->iid:$this->getIssue($id)['iid']
```

Comment:
Ok, I've ran php-cs-fixer here but I think it missed this particular case...
Updated !